### ISSUE TITLE
pmiller1@uw.edu

### ISSUE THREAD

**Issue by hippyherpdude** [ArctosDB/arctos#5485]
The new form is not working properly. At the Search screen, there are three blue pulsing dots as if it's "loading".
When I enter a Catalog number and hit "Submit query", nothing happens.
When I get into an Accession and hit "Specimen List", nothing happens.
When I go to Legacy Form, both of the above work fine.

**Comment by campmlc** [comment]
Thanks for letting us know. Can you provide a few more details as to your
browser and whether you are connecting via a work or other restricted
network?

**Comment by hippyherpdude** [comment]
Mariel,I am using Safari on a Mac computer at the Burke Museum. I am not using the museum server or any other restricted network. Like I said, the Legacy Form works fine. If there is something on our IT end such as permissions that I can ask about, please let me know. Thanks. Peter S. Miller 🐢🐍🦎🐊On Jan 15, 2023, at 15:27, Mariel Campbell ***@***.***> wrote:﻿
Thanks for letting us know. Can you provide a few more details as to your
browser and whether you are connecting via a work or other restricted
network?

**Comment by Jegelewicz** [comment]
This appears to be something other have experienced. See https://github.com/ArctosDB/arctos/issues/5420#issuecomment-1370316265 

> I have a feeling it’s not you, it’s me. ;)
> 
> OK, not me per se, but my state government network. I forwarded my email below with all the links to my personal email account and opened them all without issue on my personal laptop. And then I tried again on my work computer and some of the URLs still loaded with the blue dots. It looks like I’m having better luck with the shorter URLs. If no one else is reporting issues, then it’s probably best to assume it’s our network/firewall. If I keep running into issues I’ll start a help desk ticket with our IT guys.

**Comment by mkoo** [comment]
HI @hippyherpdude 
There's definitely some issues with Safari. I have no trouble with Firefox, Chrome, Opera but not so much with Safari which is running slower than the others. These are the settings that I finally got my Safari to run Arctos' new search UI:
- **make sure you are on the secure site**-- the url starts with **HTTPS://**
- make sure you are allowing pop-ups and disable all extensions (you can turn them back  on if you get Safari to work)
- the main clue that the webpage is not working for you is those 3 dots-- if you simply reload the page and the  3 dots just keep going then something is wrong (it's still loading the page).
I honestly am not sure what is going with Safari but the main thing is making sure you are using the secure URL https://arctos.database.museum/  
Likely Safari will not completely load any unsecure websites in an abundance of caution. I personally use FF as my main browser but often test with a bunch of others! 

Let us know how it works for you now

**Comment by jebrad** [comment]
@hippyherpdude - we were entering data on the workstation next to yours, and the searches worked fine Friday.  I will try to log in on your machine to see if I have the same problem you had. Also I will be in this Sunday so might see you there.

**Comment by mkoo** [comment]
@jebrad Please try out the tips on Safari I listed above. The search page worked for me but only if using the secured HTTPS 

@dustymc can you double check that there is a redirect for the the unsecured http please? (some browsers may still hang on to the old path but it can help if people clear caches too)

If the HTTPS page still doesnt load, then please let us know!

**Comment by dustymc** [comment]
@mkoo that's TACC, and Arctos should not be accepting any form of unencrypted traffic. I can't get there without SSL, even with Safari (which functions as expected). Can you email Chris with your bypass recipe?

**Comment by jebrad** [comment]
@hippyherpdude - Searches were working fine on your 2 computers in the Workroom: STF iMac 3 using Safari, and Herp Lab Student Mac running Windows, using Chrome.  I did not try the computer on your desk.
I got to Arctos by typing starting to type arctos into the address bar, which autofilled and appeared to take me to [httpe](https://arctos.database.museum/) but that was a cached address.  

I realize this is a separate issue but Do I really need to worry about accidentally bypassing the https and somehow getting onto an insecure version of arctos?  Or is this just something to check when eg search is not working?
Jeff
